### DISCUSSION TITLE
save on server side

### DISCUSSION THREAD

**Discussion by ibasaw** [EmulatorJS/EmulatorJS#515] [Ideas]
hi,

it would be great to have "saves" game on server side (mysql, nosql) depend on profile/user, not inside the browser storage location.

So user 1 with login1 could save his games on his profile, he could play from any device, laptop, mobile, ...
User 2 login with login2, ...

++

**Comment by ethanaobrien** [comment]
This will probably not happen. If anyone wants to create a repository that has this feature you're welcome to and I'll promote it, but I don't think this will happen.

I don't like having to communicate with the server, it can get messy (like how netplay is an insane mess) which is why this isn't supported. Also, this would most likely require server configuration or/and a specific host which makes it very dependent on what the person hosting it uses.
